app/Actions/Notifications/SendWebhookTestNotification.php

```diff
@@ -30,7 +31,7 @@ public function handle(array $webhooks)
             WebhookCall::create()
                 ->url($webhook['url'])
                 ->payload([
-                    'result_id' => fake()->uuid(),
+                    'result_id' => Str::uuid()->toString(),
```

Comment:
You shouldn't have to use `->toString()` as `->uuid()` returns a uuid v4 be default: https://laravel.com/docs/12.x/strings#method-str-uuid